Weather

SkripnikovSI opened the pull request:
Skripnikov Sergey 2538

SkripnikovSI committed:
Weather

vassiliphilippov commented:
При добавлении города приложение падает "В приложении Weather произошла ошибка"

За красивую иконку к приложению спасибо! В этом домашнем задании это пока единственное приложение с недефолтной иконкой.

SkripnikovSI committed:
fix bug

SkripnikovSI commented:
????????

22 ?????? 2013 ?., 12:19 ???????????? Vassili Philippov <
notifications@github.com> ???????:

> ??? ?????????? ?????? ?????????? ?????? "? ?????????? Weather ?????????
> ??????"
> 
> ?? ???????? ?????? ? ?????????? ???????! ? ???? ???????? ??????? ??? ????
> ???????????? ?????????? ? ??????????? ???????.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/IFMO-MobDev-2013/lesson9/pull/8#issuecomment-29056044
> .

SkripnikovSI committed:
Weather2 Beta

Using another API. So now I haven't bugs by API. Haven't changed UI yet
for new API and using the old.

thevery commented on Weather/src/ru/ifmo/ctddev/skripnikov/Weather/WeatherView.java:
```diff
@@ -0,0 +1,56 @@
+        super(context, attr);
+    }
+
+    protected int getIconId(String id) {
```
`getResources().getValue()` will make your code much simpler and cleaner

SkripnikovSI committed:
made my code much simpler and cleaner

SkripnikovSI committed:
much cleaner code in Weather2

vassiliphilippov commented:
ОК
8 баллов
